invenio_app_ils/document_requests/loaders/jsonschemas/document_request.py

```diff
@@ -53,6 +53,7 @@ class Meta:
     issn = SanitizedHTML()
     issue = SanitizedHTML()
     journal_title = SanitizedHTML()
+    medium = fields.Str(required=True)
```

Comment:
will we have a default in the frontend for the medium? It would be one less click for the UX ;)

Reply:
Yes, seems logical to me, maybe put "Paper" as default, since I suppose that most people who will request it want it in paper. WDYT?

Reply:
👍 yes, it makes sense